doc/rifle.1

```diff
@@ -1,4 +1,4 @@
-.\" Automatically generated by Pod::Man 4.10 (Pod::Simple 3.35)
```

Comment:
You didn't change `rifle.pod` so please amend it out of the commit. (Check out the previous version and amend the commit.) We get lots of unnecessary merge conflicts on the generated docs.

Reply:
Still learning a lot about using git and I think I've done it correctly
```
git checkout /doc/rifle.1 origin/master
git commit --amend
```
git now says  `Your branch and 'origin/stborderw3mfix' have diverged,`  
Doing a force push to origin should be fine, right?

Reply:
Yes, it's fine in this case because no one else is working on your fork. (At least I assume so.) You'll have to do the same for all the commits that had the file btw, not sure it was only in the last one.

Reply:
Running a `git diff` between master and my updated branch doesn't mention rifle files anymore so we're good.